src/mlpack/methods/decision_tree/gain_functions/sse_gain.hpp

```diff
@@ -66,8 +66,15 @@ class SSELoss
    * @param begin The begin index to calculate gain.
    * @param end The end index to calculate gain.
    */
-  double Evaluate(const size_t begin, const size_t end)
+  template<bool UseWeights, typename MatType, typename WeightVecType>
+  double Evaluate(const MatType& input, 
+                  const WeightVecType& /* weights */,
+                  const size_t begin, 
+                  const size_t end)
```

Comment:
Is this a `FitnessFunction`?  It doesn't seem to match the necessary API in the documentation.

Reply:
Or is it only for [`DecisionTreeRegressor`](https://www.mlpack.org/doc/user/methods/decision_tree_regressor.html#advanced-functionality-template-parameters)?